docs/2-tutorials/2-develop/6-push-notifications.html.md

```diff
@@ -2,7 +2,164 @@
 title: Push Notifications
 url: develop/push-notifications
 layout: subpage
-write: false
 ---
 
-Push Notifications content here
+You can use the **PhoneGap Developer App** paired with the **PhoneGap CLI** to immediately preview your app on a device without installing platform SDKs, registering devices, or compiling code. The PhoneGap CLI starts a small web server to host your project and returns the server address for you to pair with from the PhoneGap Developer App running on your mobile device.
+
+As of **PhoneGap Developer App** release 1.6.2 and **PhoneGap CLI** release 6.2.0 you can now test push notification functionality without needing to setup a GCM project for Android or an Apple push certification for iOS.
+
+<div class="alert--warning">Double check to ensure you're running your device and computer on the same network before continuing. </div>
+
+### Create Default PhoneGap Push Project
+
+1. Enter the following command from your terminal:
+
+		$ phonegap create myApp --template phonegap-template-push
```

Comment:
We should include the template in the `phonegap template list` command add a shortened name, such as _push_.